components/inputs/input-textarea.js

```diff
@@ -224,6 +250,9 @@ class InputTextArea extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))
 		if (this.label && this.labelHidden) {
 			return this.label;
 		}
+		if (this.hasAttribute('aria-label')) {
```

Comment:
Should this be added to the readme?

Reply:
We could... although I figured I'd leave it out though since we actually want people to use `label` and `label-hidden`.  This code is really just so existing usages don't break a11y.

Reply:
I'll add a comment here though so it's clear why this is here.

Reply:
That makes sense to me. Sounds good